### PR TITLE
[TACHYON-794] Added test case for MasterWorkerInfo

### PR DESCRIPTION
**Pull request by yupeng9**
https://tachyon.atlassian.net/browse/TACHYON-794

### CODE REVIEW

File: servers/src/test/java/tachyon/master/next/block/meta/MasterWorkerInfoTest.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+import tachyon.Constants;
+import tachyon.thrift.NetAddress;
+import tachyon.thrift.WorkerInfo;
+
+/**
+ * Unit tests for MasterWorkerInfo.
+ */
+public final class MasterWorkerInfoTest {
+  private MasterWorkerInfo mInfo;
```
Comment (gpang) [inline servers/src/test/java/tachyon/master/next/block/meta/MasterWorkerInfoTest.java]: can you put this after all the static finals?

**Comment by gpang** [comment]
Just a minor comment, but looks good! Thanks!

File: servers/src/test/java/tachyon/master/next/block/meta/MasterWorkerInfoTest.java (hunk 1 above)
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/next/block/meta/MasterWorkerInfoTest.java]: done

**Comment by gpang** [comment]
LGTM

**Comment by uronce-cc** [comment]
+1 LGTM

**Pull request merged**